test/smoke/dashCore.js

```diff
@@ -1,18 +1,116 @@
-describe('DashCore', () => {
-  describe('All nodes', () => {
-    // `subversion` from `getNetworkInfo` response.
-    it('should have correct network type');
-
-    // The number of blocks should be almost the same (-3/+3) and block hash of particular
-    // block height should be the same. Using `blocks` and `bestblockhash` from `GetBlockChainInfo`.
-    it('should propagate blocks');
+const RpcClient = require('@dashevo/dashd-rpc/promise');
+const SSH = require('simple-ssh');
+const fs = require('fs');
+
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
+
+describe('All nodes', () => {
+  networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
```

Comment:
We should iterate over all dashd nodes: https://github.com/dashpay/dash-network-deploy/pull/30#discussion_r217921683

Reply:
fixed